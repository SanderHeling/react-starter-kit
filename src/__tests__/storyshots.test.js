import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({ storyKindRegex: /^((?!.*?Modal).)*$/ });
