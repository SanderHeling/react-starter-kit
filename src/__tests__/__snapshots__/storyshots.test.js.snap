// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button with content 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <div>
    <div
      style={Object {}}
    >
      <div
        className="Button "
        onClick={[Function]}
        role="button"
        tabIndex={0}
      >
        Button
      </div>
    </div>
    <div
      style={undefined}
    >
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "backgroundColor": "#fff",
            "border": "1px solid #eee",
            "borderRadius": "2px",
            "boxShadow": "0px 2px 3px rgba(0, 0, 0, 0.05)",
            "color": "#444",
            "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
            "marginTop": "50px",
            "padding": "20px 40px 40px",
          }
        }
      >
        
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Story Source
          </h1>
          <pre
            style={
              Object {
                "backgroundColor": "#fafafa",
                "fontFamily": "Menlo, Monaco, \\"Courier New\\", monospace",
                "fontSize": ".88em",
                "lineHeight": 1.5,
                "overflowX": "scroll",
                "padding": ".5rem",
              }
            }
          >
            <div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  &lt;
                  Button
                </span>
                <span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      onClick
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        <span>
                          <span
                            style={
                              Object {
                                "color": "#170",
                              }
                            }
                          >
                            clicked()
                          </span>
                        </span>
                      </span>
                    </span>
                    
                  </span>
                </span>
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  &gt;
                </span>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 33,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  Button
                </span>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  &lt;/
                  Button
                  &gt;
                </span>
              </div>
            </div>
          </pre>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Prop Types
          </h1>
          <div>
            <h2
              style={
                Object {
                  "margin": "20px 0 0 0",
                }
              }
            >
              "
              Button
              " Component
            </h2>
            <table
              style={
                Object {
                  "borderCollapse": "separate",
                  "borderSpacing": "10px 5px",
                  "marginLeft": -10,
                }
              }
            >
              <thead>
                <tr>
                  <th>
                    property
                  </th>
                  <th>
                    propType
                  </th>
                  <th>
                    required
                  </th>
                  <th>
                    default
                  </th>
                  <th>
                    description
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    className
                  </td>
                  <td>
                    string
                  </td>
                  <td>
                    no
                  </td>
                  <td>
                    <span
                      style={
                        Object {
                          "color": "#22a",
                          "wordBreak": "break-word",
                        }
                      }
                    >
                      "
                      
                      "
                    </span>
                  </td>
                  <td />
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Button with custom class 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <div>
    <div
      style={Object {}}
    >
      <div
        className="Button customClass"
        onClick={[Function]}
        role="button"
        tabIndex={0}
      >
        Button
      </div>
    </div>
    <div
      style={undefined}
    >
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "backgroundColor": "#fff",
            "border": "1px solid #eee",
            "borderRadius": "2px",
            "boxShadow": "0px 2px 3px rgba(0, 0, 0, 0.05)",
            "color": "#444",
            "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
            "marginTop": "50px",
            "padding": "20px 40px 40px",
          }
        }
      >
        
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Story Source
          </h1>
          <pre
            style={
              Object {
                "backgroundColor": "#fafafa",
                "fontFamily": "Menlo, Monaco, \\"Courier New\\", monospace",
                "fontSize": ".88em",
                "lineHeight": 1.5,
                "overflowX": "scroll",
                "padding": ".5rem",
              }
            }
          >
            <div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  &lt;
                  Button
                </span>
                <span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      onClick
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        <span>
                          <span
                            style={
                              Object {
                                "color": "#170",
                              }
                            }
                          >
                            clicked()
                          </span>
                        </span>
                      </span>
                    </span>
                  </span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      className
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        <span
                          style={
                            Object {
                              "color": "#22a",
                              "wordBreak": "break-word",
                            }
                          }
                        >
                          "
                          customClass
                          "
                        </span>
                      </span>
                    </span>
                    
                  </span>
                </span>
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  &gt;
                </span>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 33,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  Button
                </span>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#777",
                    }
                  }
                >
                  &lt;/
                  Button
                  &gt;
                </span>
              </div>
            </div>
          </pre>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Prop Types
          </h1>
          <div>
            <h2
              style={
                Object {
                  "margin": "20px 0 0 0",
                }
              }
            >
              "
              Button
              " Component
            </h2>
            <table
              style={
                Object {
                  "borderCollapse": "separate",
                  "borderSpacing": "10px 5px",
                  "marginLeft": -10,
                }
              }
            >
              <thead>
                <tr>
                  <th>
                    property
                  </th>
                  <th>
                    propType
                  </th>
                  <th>
                    required
                  </th>
                  <th>
                    default
                  </th>
                  <th>
                    description
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    className
                  </td>
                  <td>
                    string
                  </td>
                  <td>
                    no
                  </td>
                  <td>
                    <span
                      style={
                        Object {
                          "color": "#22a",
                          "wordBreak": "break-word",
                        }
                      }
                    >
                      "
                      
                      "
                    </span>
                  </td>
                  <td />
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
